draft-bormann-t2trg-deref-id.md

```diff
@@ -222,21 +231,79 @@ than that certain content needs to be offered there (potentially
 presenting non-trivial loads, some mechanisms needed to update that
 information, and legal liabilities that are hard to assess).
 
+Breakage due to incompatible changes
+------------------------------------
+
+Dereferencing an identifier may produce different representations over time.
+While these changes may be intentional and beneficial
+(e.g. because terms are compatibly added to a resource describing terms that are evolved together {{COOL}}),
+they can also cause breakage in applications
+that previously dereferenced the identifier successfully:
+
+* There can be errors in the representation introduced by the change.
+* The operator and the consumer may disagree about what constitutes a compatible change.
+* An updated representation may exceed the consumer's capabilities,
+  e.g. not fitting in an allocated buffer.
+* Even without intended changes to the representation,
+  changes to the channel may exclude certain consumers.
+  For example, the operator's web server may cease to accept the cipher suites implemented in the consumer.
+* When the operator's services are compromised,
+  there may be malicious changes in the representation.
+
 Redirect ambiguities
 --------------------
 
 Dereferencing an identifier may involve following some redirections;
 whether that following is actually implied, or desired (or even
 desirable) is rarely being discussed.
 
+Other pitfalls
+--------------
+
+Denial of service attacks are discussed in {{seccons}}.
+Privacy implications, in particular around single-use identifiers, are discussed in {{privcons}}
+
+Usage patterns between dereferencing and precise matching
+=========================================================
+
+Consumers do not face a binary choice between dereferencing dereferencable identifiers and treating them as opaque.
+The space between those extremes is continuous.
+Some points on that way are these:
```

Comment:
?

Reply:
I've been looking for a way to phrase "this is a continuous path, and we're not going through every turn and corner because #2 listed 12 thereof, but here  are enough landmarks that you see the full path"; will try to rephrase.